editor/store/selectors.js

```diff
@@ -353,7 +354,7 @@ export function getEditedPostExcerpt( state ) {
  * @return {string} Preview URL.
  */
 export function getEditedPostPreviewLink( state ) {
-	return getCurrentPost( state ).preview_link || null;
+	return getEditedPostAttribute( state, 'preview_link' );
```

Comment:
I think you can use `getEditedPostAttribute` directly to avoid exposing another selector.

Reply:
Oh. I get what you're saying. `getEditedPostPreviewLink()` already existed in #6882 though. What's our policy on removing selectors?